velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -891,6 +891,14 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
   // 8 hours ahead UTC.
   setQueryTimeZone("Asia/Shanghai");
 
+#ifdef NDEBUG
```

Comment:
The max value of seconds allowed in Spark is `INT64_MAX/1'000'000`, so this limitation could be removed if we avoid creating invalid timestamp from the unix_timestamp function, seeing https://github.com/facebookincubator/velox/issues/9904. To use `INT64_MAX/1'000'000` as seconds, below overflow should also be reproducible.
> Timepoint is outside of supported year range: [-32767, 32767]

Reply:
+1. It would be great if there was a way to ensure this from the Spark functions themselves, since the Timestamp class is shared with Presto and others.